.github/workflows/release.yml

```diff
@@ -42,13 +42,14 @@ jobs:
           repo: "rsbuild-ecosystem-ci"
           workflow_file_name: "ecosystem-ci-selected.yml"
           github_token: ${{ secrets.REPO_SCOPED_TOKEN }}
-          ref: ${{ github.event.inputs.branch }}
+          ref: "main"
```

Comment:
Why use main instead of the input branch here

Reply:
It's the rsbuild-ecosystem-ci's reference of the workflow run, not rsbuild ref. the input branch should be passed in `client_payload`.
https://github.com/convictional/trigger-workflow-and-wait?tab=readme-ov-file#arguments

Reply:
get